lib/rust/ensogl/pack/js/src/runner/index.ts

```diff
@@ -175,6 +177,71 @@ class Scheduler {
     }
 }
 
+// =========================
+// === showConfigOptions ===
+// =========================
+
+export function showConfigOptions(config: config.Options, unknownOptions?: string[]) {
+    logger.log('Showing config options help screen.')
+    let msg = ''
+    if (unknownOptions) {
+        const optionLabel = unknownOptions.length > 1 ? 'options' : 'option'
+        msg = `Unknown config ${optionLabel}: ${unknownOptions.map(t => `'${t}'`).join(', ')}. `
+    }
+    const sectionsData: [string, string, debug.HelpScreenEntry[]][] = Object.entries(
```

Comment:
Is this code moved or changed as well?

Reply:
only moved. it needs to be accessed by `content/index.ts` so it shouldn't be an instance method. an alternative is just making it a static method, not sure whether it'd make sense for `content/index.ts` to do `app.App.showConfigOptions` though

Reply:
The current solution is perfect. I asked only to know if I should review the content or not :)